gnovm/cmd/gno/test.go

```diff
@@ -394,6 +410,18 @@ func runTestFiles(
 		if rep.Output != "" && (verbose || rep.Failed) {
 			io.ErrPrintfln("output: %s", rep.Output)
 		}
+
+		if printRuntimeMetrics {
+			imports := m.Store.NumMemPackages() - numPackagesBefore - 1
+			// XXX: pretty print big numbers
+			// XXX: store changes
+			// XXX: total alloc
+			// XXX: max mem consumption
+			io.ErrPrintfln("---       runtime: cycle=%dk imports=%d allocs=TODO memory=TODO store=TODO",
+				m.Cycles/1000,
```

Comment:
why divided by 1k?

Reply:
I would say for readability. 

@moul we have this indirect dependency https://github.com/dustin/go-humanize that can be used to make large number more human readable.